products/mozilla_org/test_mozilla_org.md

```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+* For alternative message, change URL to [Scene2](https://www.mozilla.org/en-US/firefox/new/?scene=2)
+* To fully review the Firefox download page messages, tests must go through following scenarios:
+.. * Desktop: Windows, macOS, Linux
+.. * Mobile: Android and iOS
+.. * Updating from older version of Firefox to the latest vs. downloading for the first time
+
+### [firefox/sendto.lang](https://www-dev.allizom.org/styleguide/docs/send-to-device/)
+* Enter a mobile phone number, a new page will be generated, suggesting the next action to take
+* Enter a phone number in the wrong phone number format. This will prompt an error message
+
+### [firefox/sync.lang](https://www.mozilla.org/en-US/firefox/sync/) 
+
+### firefox/tracking-protection-tour.lang
+* Start a new browser by clicking on the Firefox icon, select New Private Window on the menu list
+* Once the window is launched, click on **See how it works** button
+* You are taken [step one](https://www.mozilla.org/en-US/firefox/51.0.1/tracking-protection/start/?step=1)
+* Click the **Next** button, it takes you steps [two](https://www.mozilla.org/en-US/firefox/51.0.1/tracking-protection/start/?step=2) and [three](https://www.mozilla.org/en-US/firefox/51.0.1/tracking-protection/start/?step=3)
+* In step [three](https://www.mozilla.org/en-US/firefox/51.0.1/tracking-protection/start/?step=3), an extra text box appears 
+* Click on **Disable protection for this session** button to generate a new "step three" message  
+* Click on **Enable protection** button to toggle between "enable" and "disable" protection messages
+* Click on **Got it!** button to get to the next page
+* Click on **Restart tour** button to go through the above steps  
+
+### [mozorg/about/history.lang](https://www.mozilla.org/about/history/)
+* The slides should automatically scroll to the next in about 10 seconds or so 
+* If it doesn't or it stops, click the next or previous arrows to go to the next slide
+* Be sure to cycle to through until you see the first slide to ensure you have covered all the slides
+* Text fits inside the box
+ 
+### [mozorg/about/manifesto.lang](https://www.mozilla.org/en-US/about/manifesto/)
+You can check out the manifestos without going to the slide mode. However, only in the slide mode, all the localized content can be reviewed.
+* Click on the first manifesto
+* Click on the “next” and the “previous” arrows to move from one slide to the next
+* Check on text layout, alignment and wrapping. Box size should be adjustable to fit all the content
+
+### [mozorg/contribute/signup.lang](https://www.mozilla.org/contribute/signup/)
+There are different ways to access the signup dialog box above the footer area.  This is from the Contribute page.  
+* Click on any of the six icons or the links below each of them, you will be prompt to the Sign Up section
+* Leave the name field empty, click on the **Sign Up Now** button. A text bubble will appear
```

Comment:
Note that this is the sign-up for the newsletter, not for the contribute page. That's available even at the bottom of the home page.

The error bubbles come from Firefox, e.g. try signing up at the bottom of the Italian page, the warning will be in English https://www.mozilla.org/it/

Personally I would drop this section